pkg/roles/README.md

```diff
@@ -0,0 +1,15 @@
+# roles
```

Comment:
Planning to have APIs housed in this package eventually, but for now it just holds the roles types.